app/views/katello/api/v2/packages/backend.json.rabl

```diff
@@ -5,3 +5,8 @@ attributes :children, :checksumtype, :size, :url, :build_time, :summary, :group,
 node :human_readable_size do |package|
   number_to_human_size(package.size) if package.size
 end
+
+node :human_readable_time do |package|
```

Comment:
Let's change this to UTC time, and the view can turn that into human readable time, like this:

here:
```
node :build_time_utc do |package|                                                                                                                                                             
  Time.at(package.build_time).to_datetime if package.build_time                                                                                                                               
end
```

in the view:

```
<dd><long-date-time date="package.build_time_utc" /></dd>
```

Reply:
@jturel Thank you for the suggestion. I have updated the mentioned changes.